README.md

```diff
@@ -10,10 +10,14 @@ Konstraint is a CLI tool to assist with the creation and management of templates
 ## Installation
 
 ```text
-GO111MODULE=on go get github.com/plexsystems/konstraint
+go install github.com/plexsystems/konstraint
```

Comment:
When I attempted this, I received the following error.

```
go install: version is required when current directory is not in a module
	Try 'go install github.com/plexsystems/konstraint@latest' to install the latest version
```

I was able to successfully install by adding the `@latest` tag as the error suggested.

Reply:
change made.